sdk/iothub/azure-resourcemanager-iothub/CHANGELOG.md

```diff
@@ -1,9 +1,137 @@
 # Release History
 
-## 1.0.0-beta.2 (2021-05-13)
+## 1.0.0-beta.2 (2021-05-14)
 
+- Azure Resource Manager IotHub client library for Java. This package contains Microsoft Azure SDK for IotHub Management SDK. Use this API to manage the IoT hubs in your Azure subscription. Package tag package-2021-03. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
```

Comment:
This version has both 2021-3 and 2020-03?

Reply:
Dealing with some merge conflicts from just releasing the 2020-03 API version, so I'm going to close this PR and regen now that the 2020-03 API version release is complete